Sources/StreamVideo/Call.swift

```diff
@@ -1071,4 +1109,38 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
             log.debug("NoiseCancellationSettings updated. No action!")
         }
     }
+
+    /// Handles updates to transcription settings.
+    /// - Parameter value: The updated `TranscriptionSettings` value.
+    private func didUpdate(_ value: TranscriptionSettings?) {
+        guard let value else {
+            log.debug("TranscriptionSettings updated. No action!")
+            return
+        }
+
+        switch value.mode {
+        case .available:
+            log.debug("TranscriptionSettings updated with mode:\(value.mode).")
+        case .disabled:
+            /// Deactivate noiseCancellationFilter if mode is disabled and the noiseCancellation
+            /// audioFilter is currently active.
+            log.debug("TranscriptionSettings updated with mode:\(value.mode). Will deactivate transcriptions.")
+            Task {
+                do {
+                    try await stopTranscriptions()
+                } catch {
+                    log.error(error)
+                }
+            }
+        case .autoOn:
+            log.debug("TranscriptionSettings updated with mode:\(value.mode). Will activate transcriptions.")
+            Task {
+                do {
+                    try await startTranscriptions()
```

Comment:
We should check if we're already subscribing first.

Reply:
If we are already transcribing the mode will be available (i assume)